tools/cmake/macros_public.cmake

```diff
@@ -530,7 +530,7 @@ function(ua_generate_nodeset_and_datatypes)
 
     set(options INTERNAL AUTOLOAD)
     set(oneValueArgs NAME FILE_NS FILE_CSV FILE_BSD OUTPUT_DIR TARGET_PREFIX BLACKLIST)
-    set(multiValueArgs DEPENDS IMPORT_BSD NAMESPACE_MAP)
```

Comment:
The argument is being used elsewhere in the file. They should all be removed.

Reply:
Thx, removed.